templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/WelcomeCommunities.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="modal ob-font">
+    <b-container fluid class="center">
+      <b-row align-h="center">
+        <h1 style="max-width: 390px;">
+          Welcome to your circle of connections!
+        </h1>
+      </b-row>
+
+      <b-row align-h="center">
+        <h4 class="ob-secondary">
+          Let's start with you — what communities are you a part of?
+        </h4>
+      </b-row>
+
+      <b-row align-h="center">
+        <h5 class="ob-secondary">Here are a few ideas to get you started.</h5>
+      </b-row>
```

Comment:
Again, it's better to avoid using heading elements just for styling purposes. Use (S)CSS instead